graylog2-server/src/main/java/org/graylog2/streams/StreamRuleService.java

```diff
@@ -32,10 +32,12 @@
 
     StreamRule create(Map<String, Object> data);
 
-    StreamRule create(String streamid, CreateStreamRuleRequest request);
+    StreamRule create(String streamId, CreateStreamRuleRequest request);
 
     List<StreamRule> loadForStreamId(String streamId) throws NotFoundException;
 
+    Map<String, List<StreamRule>> loadForStreamIds(List<String> streamIds);
```

Comment:
@bjoernhaeuser True and please keep that.

The method parameter `streamIds` should be a `Collection<String>`, though.